lib/gym/package_command_generator.rb

```diff
@@ -45,6 +56,48 @@ def ipa_path
       def dsym_path
         Dir[BuildCommandGenerator.archive_path + "/**/*.dsym"].last
       end
+
+      private
+
+      # Fix PackageApplication Perl script by Xcode to create the IPA from the archive
+      def patch_package_application
+        require 'fileutils'
+
+        # Initialization
+        developer_dir = `xcode-select --print-path`.strip
+        patched_package_application_path = File.join("/tmp", "PackageApplication4Gym")
+        # Remove any previous patched PackageApplication
+        FileUtils.rm patched_package_application_path
+
+        Dir.mktmpdir do |tmpdir|
+          # Check current PackageApplication MD5
+          require 'digest'
+
+          expected_md5 = File.read("lib/assets/package_application_patches/PackageApplication_MD5")
```

Comment:
consider creating a constant for the path of lib/assets/package_application_patches.

We might need to support multiple directories one day, e.g. if Apple fixes some issues in a future Xcode version.